crates/subspace-core-primitives/src/lib.rs

```diff
@@ -215,6 +222,144 @@ impl PosProof {
     pub const SIZE: usize = 17 * 8;
 }
 
+/// Proof of time key(input to the encryption).
+#[derive(Debug, Copy, Clone, Encode, Decode)]
+pub struct PotKey(PotBytes);
+
+impl PotKey {
+    /// Returns the key bytes.
+    #[inline]
+    pub fn bytes(&self) -> PotBytes {
+        self.0
+    }
+}
+
+impl From<PotBytes> for PotKey {
+    #[inline]
+    fn from(value: PotBytes) -> Self {
+        Self(value)
+    }
+}
+
+/// Proof of time seed (input to the encryption).
+#[derive(Debug, Copy, Clone, Encode, Decode)]
+pub struct PotSeed(PotBytes);
```

Comment:
How is seed different from key?

Reply:
Key is artifact from AES, since it takes 2 arguments. In contrast to normal AES in our case key has to be set to some publicly known value at chain start, and later updated with every iteration like seed.
The notion of key will most certainly become obsolete when we update to proper VDF.